test/lib/ansible_test/_data/requirements/integration.cloud.vcenter.txt

```diff
@@ -1 +1,2 @@
 pyvmomi
+git+https://github.com/vmware/vsphere-automation-sdk-python.git
```

Comment:
Is this SDK not available on pypi?

Reply:
Nah, I asked myself the very same question: https://github.com/vmware/vsphere-automation-sdk-python/issues/38

Reply:
No this is not available on PyPi :(